### ISSUE TITLE
Call to a member function getBaseUrl() on null

### ISSUE THREAD

**Issue by sander-lameco** [putyourlightson/craft-blitz#581]
We have a site running Blitz Cache with multiple sites. One of those sites is not yet enabled and we try to view the frontend using the siteToken via the CMS. When disabling Blitz Cache there is not error, when enabled the error comes up.
The BaseUrl of the site is not empty, it uses a .env variable as a value.

# Steps to reproduce the issue
1. Create a site but don't enable it.
2. Create a section with a `preview target` set.
3. Create an entry in the section and save it.
4. Open the preview target via view -> preview target.

You should now get the error 
```
Call to a member function getBaseUrl() on null
```

```
1. in /%%/%%/%%/%%/vendor/putyourlightson/craft-blitz/src/drivers/storage/FileStorage.php
```

```
Line #297: $siteUrl = Craft::getAlias($site->getBaseUrl());
```

# A clear and concise description of what you expected to happen.
I would expect to be able to view the page

#The plugin and Craft version numbers
Blitz Cache: `4.8.0`
Craft CMS: `4.5.10`

**Comment by bencroker** [comment]
Thanks for reporting this! Fixed in https://github.com/putyourlightson/craft-blitz/commit/909d274ec0f74845b130327f995d6b79a637ee64 for the next release.

You can test this by running `composer require "putyourlightson/craft-blitz:dev-develop as 4.8.1"`.

**Comment by bencroker** [comment]
Released in versions [4.9.0](https://github.com/putyourlightson/craft-blitz/releases/tag/4.9.0) and [3.13.0](https://github.com/putyourlightson/craft-blitz/releases/tag/3.13.0).

**Comment by sander-lameco** [comment]
Just to confirm: its working. Thanks!
